web/libs/editor/src/regions/TextAreaRegion/TextAreaRegion.module.scss

```diff
@@ -1,47 +1,69 @@
 .mark {
-  cursor: pointer;
-  background-color: #f6ffed;
-  border: 1px solid #b7eb8f;
-  border-radius: 5px;
-  margin: 0;
-  padding: 0.4em 1em;
+  background-color: var(--surface-surface);
+  border: 1px solid var(--surface-border);
+  border-radius: var(--radius-small);
+  margin: 0 var(--spacing-small) 0 0 !important;
+  padding: var(--spacing-small) var(--spacing-medium);
   white-space: pre-line;
+  box-shadow: 0px 4px 12px 0px var(--black_4), 0px 1px 2px 0px var(--black_10);
+  flex-grow: 1;
 
-  &.selected {
-    border: 1px solid red;
+  &[class*=selected] {
+    border: 1px solid var(--grape_500);
+    background-color: var(--primary-bright-secondary);
   }
 
-  &.highlighted {
-    border: 1px dashed #00aeff;
+  &[class*=highlighted] {
+    border: 1px dashed var(--grape_500);
+    background-color: var(--primary-bright-secondary);
   }
 
   &.relation {
     cursor: crosshair;
   }
 }
 
-.delete {
-  color: #1890ff;
-  padding-top: 0.5em;
-  padding-left: 1em;
-  line-height: 1.8em;
-}
 
 .row {
   display: flex;
-  margin-bottom: 0.5em;
+  margin-bottom: var(--spacing-medium);
+  flex-wrap: nowrap;
+  align-items: center;
+  gap: var(--spacing-small);
+
+  > button {
+    display: flex;
+    border-radius: var(--radius-small);
+    padding: 0 !important;
+    margin: 0 !important;
+    flex: 0 0 32px;
+    height: 32px;
+
+    &:hover {
+      background-color: var(--sand_200);
+    }
+
+    &[class$=delete], &[class*=delete][class*=tooltip-open] {
+      color: var(--danger_color) !important;
+
+      &:hover {
+        background: var(--red_1) !important;
+        color: var(--danger_color);
+      }
+    }
+  }
```

Comment:
I think the Button component has styles that may get close to what is needed already, and generally that is where we would be wanting to capture that styling. Let's check the Button component and see what we got there, we may want to add/modify the classes there.

What you have here is a ghost, icon button (hover effect, square only icon).